src/pages/updates/v10-migration/design.mdx

```diff
@@ -1,145 +1,147 @@
+Designers can subscribe to the design kit library via Sketch Cloud, and updates to the contents of the kit will be pushed to subscribers as they become available.
 
 #### Icons, pictograms, and the full IBM color palette
 
-To access the IBM Design Language icons, pictograms, and colors, subscribe to the IBM Design Language Kit via Sketch Cloud. **Please note that all icons in the Carbon kit are now imported from the IBM Design Language library**.
+To access the IBM Design Language icons, pictograms, and colors, subscribe to the [IBM Design Language libraries](/get-started/design/sketch#get-the-kit).
 
 #### Migrating to v10
 
-Upgrading to the v10 kit will not automatically change any files created in previous versions of the kit. The migration of existing v9 layouts will be a manual process for designers.
+Upgrading to the v10 kit will not automatically change any files created in previous versions of the kit. Swapping v9 symbols for v10 symbols is a manual process.
 
 Although libraries from multiple versions of Carbon can exist in Sketch at the same time, you should avoid using v10 symbols in an app designed with v9.
 
 ### Setting up the kit
 
-The new method for using the kit is via Sketch Cloud subscription. Head over to our Getting started guide to set up the new Sketch libraries!
+Head over to the [Get started guide](/get-started/design/sketch#get-the-kit) to set up the new Sketch libraries.
+
+## Elements
+
+### Themes and color
+
+Carbon offers four color themes. See the [theming guidelines](/guidelines/color/overview#themes) to learn how to use the themes, and visit the color migration guide to learn more about color tokens added in v10.
 
 <Row>
 <Column colLg={4} colMd={4} noGutterSm>
 <ResourceCard
-    subTitle="Getting started"
-    href="/get-started/design"
+    subTitle="Migrating color"
+    href="https://github.com/carbon-design-system/carbon/blob/master/docs/migration/10.x-color.md"
     >
 
-<MdxIcon name="codesandbox" />)
+<MdxIcon name="github" />
 
   </ResourceCard>
 </Column>
 </Row>
 
-## Elements
-
-#### Themes & color
-
-Carbon now offers four preset visual themes for IBM products, offering more options than ever for accessible components. See [themes guidelines](/guidelines/themes) for detailed info on using themes in Carbon v10.
-
-Carbon color tokens also have been updated with v10 to include several new and updated color tokens.
-
-[v10 Color migration](https://github.com/carbon-design-system/carbon/blob/master/docs/migration/10.x-color.md)
+### Icons
 
-#### Icons
+The [icon library](/guidelines/icons/library) was updated and expanded with v10. The full library of icons and pictograms is available via the IBM Design Language icon [Sketch library](/get-started/design/sketch#get-the-kit).
```

Comment:
Same as previous comment, anchor links make its confusing where the user ends up landing not where the icon kit actual is. I think we should link to the icons page itself, instead of the getting started page, that way it will be more in context. https://www.carbondesignsystem.com/guidelines/icons/usage#resources